app/code/Magento/Catalog/Model/Entity/Attribute.php

```diff
@@ -111,7 +112,8 @@ public function __construct(
         LockValidatorInterface $lockValidator,
         \Magento\Framework\Model\ResourceModel\AbstractResource $resource = null,
         \Magento\Framework\Data\Collection\AbstractDb $resourceCollection = null,
-        array $data = []
+        array $data = [],
+        \Magento\Eav\Model\Entity\Attribute\FrontendLabelFactory $frontendLabelFactory
```

Comment:
Hi, @facundocapua, due to [Magento backward-compatible guide](http://devdocs.magento.com/guides/v2.2/contributor-guide/backward-compatible-development/) we can't add new required parameters to the constructor.
Please do the same as you do in `app/code/Magento/Catalog/Model/ResourceModel/Eav/Attribute.php`

Reply:
Sorry, I forgot to change that.

Thanks for pointing that out!